### DISCUSSION TITLE
Cameras/Exclusion zones

### DISCUSSION THREAD

**Discussion by JoeyJoe6** [djdd87/SynoAI#186] [General]
I have a camera at the end of my long driveway (road_camera). I added in exclusion zones so I would not flag alerts for passing vehicles. My secondary goal however is to be alert when anything else in the field of view is detected, but since I have exclusion zones it will not trigger. 

Since SynoAI does not like me using the same camera in the appsettings for different AI checks, I have created an additional camera with the same feed as road_camera and added into my appsettings.json to detect "people" and a few other types. However this uses an additional license. 

I am looking for any additional other suggestions on how to achieve monitoring the same camera feed. Also is there a way to detect everything minus "cars", "trucks", etc...? instead of having a very lengthy list of types?

thanks!

**Comment by djdd87** [comment]
I have a plan to add types to exclusion zones, but for the time being I don't see any way around it. Development is slow on this project currently due to work, renovating a house and family, but I'll get back to it ASAP.
